implants/lib/eldritch/src/sys/get_ip_impl.rs

```diff
@@ -0,0 +1,81 @@
+use anyhow::Result;
+use network_interface::{NetworkInterfaceConfig, NetworkInterface};
+use starlark::{values::{dict::Dict, Heap, Value}, collections::SmallMap, const_frozen_string};
+
+const UNKNOWN: &str = "UNKNOWN";
```

Comment:
🤷🏼‍♂️ For recoverable errors we've been returning an UNKNOWN string.
So far we've been saving this as a const in the function implementation in case it needs to be used multiple places.